examples/foreach/foreach.bal

```diff
@@ -51,10 +52,10 @@ public function main() {
     }
 
     io:println("\nIterating XML :");
-    xml book = xml `<book>
-                        <name>Sherlock Holmes</name>
-                        <author>Sir Arthur Conan Doyle</author>
-                    </book>`;
+    xmllib:Element book = <xmllib:Element> xml `<book>
+                                                    <name>Sherlock Holmes</name>
+                                                    <author>Sir Arthur Conan Doyle</author>
+                                                </book>`;
     // Iterating an XML will return each element in each iteration.
     counter = 0;
     foreach var x in book.getChildren().elements() {
```

Comment:
This is fixed in https://github.com/ballerina-platform/ballerina-lang/pull/21796/files/a7fad7f6da21e406beb2e8fa741b02381b6404cf#diff-962f5f5df0bed5b1c95c90a60c0ba262R60

Reply:
Thanks